src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_inpaint.py

```diff
@@ -1527,10 +1559,22 @@ def denoising_value_valid(dnv):
         is_strength_max = strength == 1.0
 
         # 5. Preprocess mask and image
-        init_image = self.image_processor.preprocess(image, height=height, width=width)
+        if padding_mask_crop is not None:
+            crops_coords = self.mask_processor.get_crop_region(mask_image, width, height, pad=padding_mask_crop)
```

Comment:
we don't need `height, width = self.image_processor.get_default_height_width(image, height, width)` here?

Reply:
We don't need because it already been initialized: https://github.com/huggingface/diffusers/pull/6360/files#diff-87969947ac09db062b29b1f7285246c055f9219b34b89c871884e5feee2107e7R1467-R1469